.github/workflows/pr-title.yml

```diff
@@ -25,6 +25,7 @@ jobs:
             mariadb
             mssql
             mysql
+            oracle
```

Comment:
I was wondering if we want to use oracle or oracledb here. Feel free to let me know your ideas on it

Reply:
> I was wondering if we want to use oracle or oracledb here. Feel free to let me know your ideas on it

oracle is fine for me

Reply:
I vote to always follow whatever is written here https://github.com/sequelize/sequelize/blob/v6/src/sequelize.d.ts#L172

Reply:
Makes sense to me!